wire-library/wire-swift-generator/src/main/java/com/squareup/wire/swift/SwiftGenerator.kt

```diff
@@ -569,6 +569,15 @@ class SwiftGenerator private constructor(
               }
               .build()
           )
+        } else {
```

Comment:
Can we remove the `if` altogether and get the same result? Is there any harm in having String as the super type of an empty enum?

Reply:
Unfortunately that doesn't compile: 
<img width="1174" alt="Screen Shot 2022-11-17 at 4 33 03 PM" src="https://user-images.githubusercontent.com/2643476/202589349-1f4b52c1-6f71-4936-a64b-d43216e5391a.png">

Reply:
Ah, got it.

Can we do then instead
```kotlin
.apply {
  if (theStuff.isNotEmpty()) {
    addSupertype(string)
    for (stuff in theStuff) {
      addEnumConstant(...)
    }
  }
}
```
so we can at least reuse the majority of the type definition?

Reply:
Sure, updated.